src/opyrability.py

```diff
@@ -43,9 +43,9 @@ def multimodel_rep(model: Callable[...,Union[float,np.ndarray]],
                   resolution: np.ndarray,
                   polytopic_trace: str = 'simplices',
                   perspective: str = 'outputs',
-                  plot: str = True,
-                  EDS_bound: str = None,
-                  EDS_resolution: str = None):
+                  plot: bool = True,
+                  EDS_bound = None,
+                  EDS_resolution = None):
```

Comment:
not sure if you wanted to default as 
```suggestion
                  EDS_bound: np.ndarray = None,
                  EDS_resolution: np.ndarray =  = None):
```